.gitignore

```diff
@@ -0,0 +1,19 @@
+*.log
+wp-config.php
```

Comment:
We don't need these Wordpress files and folders in this gitignore file

In fact, I don't think we need any of these?

Reply:
Totally remove .gitignore?